developers/academy/js/starter_multimodal_data/900_next_steps.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Next steps
+---
+
+import IntroNextSteps from '../_snippets/intro_next_steps.mdx';
+
+<IntroNextSteps />
```

Comment:
Hey these include links to Python courses which probably don't make a lot of sense for TS users. 

Let's edit them for TS, maybe you could think of resources to link to even if they are not in the Academy

Reply:
true, didn't notice! Added